### PR TITLE
Telegram v2

### PR DESCRIPTION
**Pull request by leddest**
- Updated getting current events
- Added creating new events

### CODE REVIEW

File: app/telegram/models.py [hunk 1]
```diff
@@ -27,6 +27,8 @@ def _get_first_name(self, data: Dict) -> str:
 
 
 class Bot:
+    MEMORY = {}
```
Comment (yammesicka) [inline app/telegram/models.py:30]: This is problematic. It's equivalent to a global variable and might create problems when working asynchronously.

File: app/telegram/handlers.py [hunk 2]
```diff
@@ -76,26 +97,145 @@ async def this_week_handler(self):
...
+            text += f'Starts on:\n{start}\n\n'
+            text += f'Ends on:\n{end}'
+            await telegram_bot.send_message(
+                chat_id=self.chat.user_id, text=text)
+            await asyncio.sleep(1)
 
-        await telegram_bot.send_message(chat_id=self.chat.user_id, text=answer)
+        return answer
+
+    async def process_new_event(self, memo_dict):
```
Comment (yammesicka) [inline app/telegram/handlers.py:131]: Split to functions

File: app/telegram/handlers.py [hunk 3]
```diff
@@ -76,26 +97,145 @@ async def this_week_handler(self):
...
+
+        answer = f"{chosen_date.strftime('%A, %B %d')}:\n"
+        await telegram_bot.send_message(
+            chat_id=self.chat.user_id, text=answer)
 
         for event in events:
-            answer += f'''
-From {event.start.strftime('%d/%m %H:%M')} \
-to {event.end.strftime('%d/%m %H:%M')}: {event.title}.\n'''
+            start = event.start.strftime("%d %b %Y %H:%M")
```
Comment (yammesicka) [inline app/telegram/handlers.py:118]: This should be a separate function :)

**Review by yammesicka — CHANGES_REQUESTED** [review]
Awesome job! Looks really cool. I left you with 1 major architecture change to do, and 2 SRP issues :)

File: app/telegram/bot.py [hunk 4]
```diff
@@ -9,3 +11,7 @@
 WEBHOOK_URL = settings.webhook_url
 
 telegram_bot = Bot(BOT_API, WEBHOOK_URL)
+
+loop = asyncio.get_event_loop()
```
Comment (yammesicka) [inline app/telegram/bot.py:15]: I think you can just `asyncio.run` the `set_webhook`

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/telegram/bot.py (hunk 4 above)
Comment (leddest) [reply app/telegram/bot.py:15]: It's impossible - asyncio.run() cannot be called from a running event loop (which started by FastAPI).

**Pull request merged**